guides/ecosystem/index.md

```diff
@@ -1,26 +1,26 @@
-# The Aries JavaScript Ecosystem
+# The Credo Ecosystem
 
-The Aries JavaScript ecosystem includes various components that are spread across multiple repositories. In this section we'll go over what these components are and how the relate to one another.
+The Credo ecosystem includes various components that are spread across multiple repositories. In this section we'll go over what these components are and how the relate to one another.
 
 ## Repositories
 
-Currently, Aries JavaScript is composed out of three separate repositories:
+Currently, Credo is composed out of three separate repositories:
 
-1. [Aries Framework JavaScript](https://github.com/hyperledger/aries-framework-javascript)
-2. [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext)
+1. [Credo](https://github.com/hyperledger/aries-framework-javascript)
+2. [Credo Extensions](https://github.com/hyperledger/aries-framework-javascript-ext)
 3. [Aries Mobile Agent React Native (Bifold)](https://github.com/hyperledger/aries-mobile-agent-react-native)
 
 ## Documentation
 
 You can find their documentation here:
 
-1. [Aries Framework JavaScript](https://credo.js.org/guides)
-2. [Aries Framework JavaScript Extensions](https://credo.js.org/guides/extensions)
+1. [Credo](https://credo.js.org/guides)
+2. [Credo Extensions](https://credo.js.org/guides/extensions)
 3. [Aries Mobile Agent React Native (Bifold)](https://github.com/hyperledger/aries-mobile-agent-react-native/blob/main/README.md)
 
-### Aries Framework JavaScript
+### Credo
 
-Aries Framework JavaScript (AFJ) is at the core of the Aries JavaScript ecosystem. AFJ provides all the functionality related to cryptography, storage, messaging and more that is required
+Aries Framework JavaScript (AFJ) is at the core of the Credo ecosystem. AFJ provides all the functionality related to cryptography, storage, messaging and more that is required
```

Comment:
I think the regex didn't work everywhere

```suggestion
Credo is at the core of the Credo ecosystem. Credo provides all the functionality related to cryptography, storage, messaging and more that is required
```

Reply:
I have intentionally skipped it where the text needs rewriting. "Credo is at the core of Credo" doens't make a lot of sense. I'll rewrite those after everything is renamed.